README.md

```diff
@@ -1 +1,56 @@
-## Math React App
\ No newline at end of file
+## Math React App
+
+>"Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to: Make simple calculations and Read a random math-related quote.
+
+## Built With
+
+- HTML, JavaScript
+- React
+- Linters:
+  - Stylelint, Lighthouse, Webhint
+  - ESLint
+
+## Live Demo 
+
+in progess 
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Install
+Have NodeJS installed and follow the next steps
+### Usage
+- `npm i` for installing dev dependecies
+- `npm run build` or `npm start` to see the project
+### Run tests
+- `npm test`
+### Deployment
+- `npm run deploy`
+
+
+## Authors
+
+👤 **Peter Boro**
+
+- GitHub: [@peterboro](https://github.com/peterboro)
+- LinkedIn: [Peter Njuguna](https://www.linkedin.com/in/peter-n-3bb940122/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- [Microverse](https://microverse.org)
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] You have added a link to your license file (MIT.md) in your readme file but I guess you forgot to add a license file in your project :smile:

1. No worries you can add this file using this [template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) :blush:

2. Also, don't forget to add **your name**, **current year**, and **app type** in the MIT.md file because most students forgot to add these things :thinking: